browser/src/control/Toolbar.js

```diff
@@ -720,6 +722,16 @@ L.Map.include({
 		}
 	},
 
+	showSnackbar: function() {
+		var timeout = 1000;
+		this.uiManager.showSnackbar('Version information has been copied', null, null, timeout);
+		var copybutton = document.querySelector('#modal-dialog-about-dialog-box-copybutton > img');
+		L.LOUtil.setImage(copybutton, 'lc_clipboard-check.svg', this._docLayer._docType);
+		setTimeout(function () {
+			L.LOUtil.setImage(copybutton, 'lc_copy.svg', this._docLayer._docType);
+		}.bind(this), timeout);
+	},
+
```

Comment:
Do we need this?:

1. maybe we could avoid having another showSnackbar function (it can be quite confusing)
2. avoid changing the showSnackbar function in UIManager and instead just close it directly from Toolbar.js . I think we are doing something like ```setTimeout(function () { instance.that.closePopover(instance.id, false); }, this.options.snackbarTimeout);``` elsewhere

Reply:
1. showSnacbar function is for code which is kind of duplicating so i have added that in showSnacbar function. **You are right i need to rename because it is bit confusing.** 

2. So the main resone i have added timout in UImanager is because i need to pass custome timeout to toolbar.js. ( Default is 10 sec in toolbar.js )

i have made some change in toolbar.js  i think we need to add a parameter timeout because for diffrent showSnackbar messages we may want diffrent timeouts to show message.